runtime/common/metering.go

```diff
@@ -134,17 +134,20 @@ func NewArrayMemoryUsages(length int) (MemoryUsage, MemoryUsage) {
 		}
 }
 
-func NewArrayAdditionalLengthUsage(originalLength, additionalLength int) MemoryUsage {
-	var newAmount uint64
-	if originalLength <= 1 {
-		newAmount = uint64(originalLength + additionalLength)
+func atreeAdditionalMemoryUsage(originalSize int) uint64 {
+	if originalSize <= 1 {
+		return uint64(originalSize + 1)
```

Comment:
@fxamacker How many nodes does an atree data structure have at minimum? Is it the case that there is always at least one `MetaDataSlab` + one` DataSlab`? Or is it possible for a root to be just a single `DataSlab` (i.e. no `MetaDataSlab`, e.g. if the container is empty or the total size of the elements is smaller than the threshold)?

Reply:
> How many nodes does an atree data structure have at minimum? 

At minimum, atree container has one DataSlab (leaf node) as root node.

> Is it the case that there is always at least one MetaDataSlab + one DataSlab? 

No, atree container can have 
- one DataSlab (leaf node) as root node, or 
- at least 3 nodes (2 DataSlabs + 1 MetadataSlabs) with MetadataSlab as root node
 
> Or is it possible for a root to be just a single DataSlab (i.e. no MetaDataSlab, e.g. if the container is empty or the total size of the elements is smaller than the threshold)?

Yes.  When total size of elements in the container is less than threshold,  DataSlab is the root node.